Tests/Acceptance/inc/locators.json

```
@@ -1,7 +1,8 @@
 {
   "checkout": {
     "nextStep": "//button[contains(@class, 'nextStep')]",
-    "paymentMethod": "payment_%s"
+    "paymentMethod": "payment_%s",
+    "piaTableId": "pia-payment-info"
```

It's not necessary to state that the locator is an ID.

```suggestion
    "piaTable": "pia-payment-info"
```

---

👍